scripts/test/Muon/MaxEntPresenter_test.py

```diff
@@ -66,11 +67,11 @@ def test_buttonWithoutRaw(self):
 
```

Comment:
With the tests involving handleMaxEntButton do you not want to test that the threads are called correctly as well? I think you could replace handleMaxEntButton with getMaxEntInput in the current tests and still pass? I agree you don't need a test for the cancel button though as whilst it has an if statement it is a trivial one.

Reply:
no, it will not call thread.start if we just call the get input. The thread.start wraps the maxEnt algorithm so start calls it

Reply:
Yep I was being blind and had missed the start thread check.